src/aiida/schedulers/plugins/sge.py

```diff
@@ -300,6 +305,8 @@ def _parse_joblist_output(self, retval, stdout, stderr):
 
         try:
             first_child = xmldata.firstChild
+            if first_child is None:
+                raise SchedulerError
```

Comment:
TODO: Not sure if this is actually possible, but we should write an error message here